app/controllers/casa_cases_controller.rb

```diff
@@ -140,12 +140,9 @@ def copy_court_orders
   def set_casa_case
     @casa_case = current_organization.casa_cases.friendly.find(params[:id])
   rescue ActiveRecord::RecordNotFound
-    if params[:action] != "show"
```

Comment:
Why is this removed?

Reply:
I discussed with Shen about this during office hours. It seemed like the idea was to return a 404 code, but it would also return a blank body. It was decided that the intention was to not allow this kind of cross-org access, so we decided to redirect to the index in all cases.

I think this exception for `show` was for the JSON request, which we now handle via `respond_to { format.json }` instead.

Reply:
Gotcha!